src/lib/button/button.ts

```diff
@@ -77,9 +77,15 @@ export class MatButton extends _MatButtonMixinBase
 
   /** Whether the button is round. */
   _isRoundButton: boolean = this._hasHostAttributes('mat-fab', 'mat-mini-fab');
+  get isRoundButton(): boolean {
```

Comment:
Instead of making these getters, could you just make the existing properties public and `readonly`? [TypeScript generates a fair amount more code for getters](https://www.typescriptlang.org/play/#src=class%20GetterExample%20%7B%0D%0A%20%20%20%20get%20something()%20%7B%0D%0A%20%20%20%20%20%20%20%20return%20true%3B%0D%0A%20%20%20%20%7D%0D%0A%7D%0D%0A%0D%0A%2F*****%2F%0D%0A%0D%0Aclass%20PropertyExample%20%7B%0D%0A%20%20%20%20readonly%20something%20%3D%20true%3B%0D%0A%7D), so it's better to avoid when you don't also need a setter.

Reply:
More than happy to, wanted to avoid what might be perceived as a breaking change.

Reply:
We consider anything prefixed with an underscore to be a private API, so it should be okay

Reply:
Done. PTAL. I only found the one if block, and the few .html references in this repo.